ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,1136 @@
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+<https://semopenalex.org/vocab#> a owl:Ontology;
+  rdfs:label "SemOpenAlex Ontology"@en;
+  rdfs:comment "The Semantic OpenAlex Ontology, described using W3C RDF Schema and the Web Ontology Language OWL."@en .
+
+<https://semopenalex.org/class/Author> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Author"@en .
+
+<https://semopenalex.org/class/Institution> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Institution"@en .
+
+<https://semopenalex.org/class/Work> a owl:Class;
+  owl:equivalentClass <http://purl.org/spar/fabio/Work>;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Work"@en .
+
+<https://semopenalex.org/class/ConceptScore> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "ConceptScore"@en .
+
+<https://semopenalex.org/class/CountsByYear> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "CountsByYear"@en .
+
+<https://semopenalex.org/class/OpenAccess> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "OpenAccess"@en .
+
+<https://semopenalex.org/class/Concept> a owl:Class;
+  rdfs:subClassOf skos:Concept;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Concept"@en .
+
+<https://semopenalex.org/class/Geo> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Geo"@en .
+
+<https://semopenalex.org/class/Publisher> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Publisher" .
+
+<https://semopenalex.org/class/AuthorPosition> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "AuthorPosition"@en .
+
+<https://semopenalex.org/class/Source> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Source"@en .
+
+<https://semopenalex.org/class/Location> a owl:Class;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "Location"@en .
+
+<http://purl.org/dc/terms/creator> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Author>;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#> .
+
+<http://purl.org/dc/terms/title> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range xsd:string;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#> .
+
+rdfs:seeAlso a owl:ObjectProperty;
+  rdfs:domain [ a owl:Class;
+      rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+      owl:unionOf (<https://semopenalex.org/class/Institution> skos:Concept)
+    ];
+  rdfs:range <http://purl.org/spar/fabio/WikipediaEntry>;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#> .
+
+skos:Concept a owl:Class;
+  rdfs:isDefinedBy <http://www.w3.org/2004/02/skos/core>, <https://semopenalex.org/vocab#>;
+  rdfs:label "Concept" .
+
+skos:broader a owl:ObjectProperty;
```

Comment:
@phaasenase do we really want to redeclare properties and clases from SKOS in this ontology?

Reply:
this triple was originally there when we received from AIFB. i would leave as it for now.